### DISCUSSION TITLE
Android Beta: Tapping back on search exits app

### DISCUSSION THREAD

**Discussion by elisealix22** [community/community#141328] [Mobile]
Hi friends 😁,

I'm using v1.182 beta and noticed when searching from home that if I open search and gesture back, it quits the app instead of going back to home.

Big fan of the new nav 💚

Elise

![Screenshot_20241013-092051.png](https://github.com/user-attachments/assets/3f31e468-2166-4783-b427-8dd6936e559b)

**Comment by stevepopovich** [comment]
Hi @elisealix22 having trouble reproducing here. Would you be interested in joining the GitHub mobile team and showing us?!?!?!?!?!

🙃 Jk 🙏🏼 filed an issue!
